src/components/__tests__/__snapshots__/navigation_desktop.test.tsx.snap

```diff
@@ -0,0 +1,121 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`The NavigationDesktop component should render 1`] = `
+<div
+  className="navigation__desktop show-sm layout layout--gutter-none layout--align-middle"
+>
+  <div
+    className="layout__item u-fit-sm"
+  >
+    <nav
+      className="navigation__container"
+      role="navigation"
```

Comment:
It is not necessary to add navigation role to a nav element.
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Navigation_Role

Reply:
updated